### PR TITLE
add new locale

### PR DESCRIPTION
**Pull request by inkhorn-ptbr**
## Description

Fixes # (adds new locale)

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I wrote unit tests for my code _(if tests is required for my changes)_
- [ ] I have made changes in `CHANGELOG.md` file

### CODE REVIEW

**Comment by inkhorn-ptbr** [comment]
New strings in Português (BR)

**Comment by tarampampam** [comment]
:+1: Great thanks for your contribution!

**Comment by inkhorn-ptbr** [comment]
you're welcome

File: public/_locales/pt_BR/messages.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  "blacklist_domains": {"message": "Lista de nomes de domínio (um por linha)"},
+  "blacklist_custom_rules": {"message": "Regras personalizadas (um por linha)"},
+  "blacklist_custom_rules_hint": {"message": "Você pode usar caracteres especiais, tais como * e ?. * coincidirá com qualquer quantidade de caracteres (ex. *google.com coincidirá com google.com, www.google.com, mail.google.com, etc.), ? coincidirá apenas com um único caractere (ex. www.?oogle.com corresponderá com www.oogle.com, www.moogle.com, www.google.com, www.woogle.com, etc.)"},
+  "previous_settings": {"message": "Configurações anteriores"},
+  "remove": {"message": "Remove"},
+  "previous_settings_hint": {"message": "Estas configurações foram usadas por você em versão anterior da extensão. Guarde-a em algum lugar ou remova-a"},
+  "save_changes": {"message": "Salvar mudanças"},
+  "error_occurred": {"message": "Ocorreu um erro"},
+  "like_this_extension": {"message": "Do you like this extension?"},
+  "give_a_star_on_github": {"message": "Give us a star on GitHub!"},
```
Comment (tarampampam) [inline public/_locales/pt_BR/messages.json:47]: @inkhorn-ptbr Could you please translate those lines too?

```suggestion
  "like_this_extension": {"message": "Você gosta desta extensão?"},
  "give_a_star_on_github": {"message": "Dê-nos uma estrela no GitHub!"},
```

**Pull request merged**